src/views/sidebar/accordion-custom-component.tsx

```diff
@@ -522,6 +523,28 @@ export const AccordionCustomComponent: FC<{ item: AccordionFolder }> = ({ item }
 
 	const dropdownItems = useFolderActions(item);
 
+	const sharedStatusIcon = useMemo(() => {
+		if (!folder.acl?.grant) {
+			return '';
+		}
+
+		const tooltipText = t('tooltip.folder_sharing_status', {
+			count: folder.acl.grant.length,
+			defaultValue_one: 'Shared with {{count}} person',
+			defaultValue: 'Shared with {{count}} people'
+		});
+
+		return (
+			<Padding left="small">
+				<Tooltip placement="right" label={tooltipText}>
```

Comment:
generally we avoid using div and any other html components directly ,you can use Row / Container from DS

Reply:
You're right, the Row does the trick too. I'll fix it